files/image_config/sysctl/sysctl-net.conf

```diff
@@ -34,6 +34,6 @@ net.ipv6.conf.all.keep_addr_on_down=1
 net.ipv6.conf.eth0.keep_addr_on_down=1
 net.ipv4.tcp_l3mdev_accept=1
 net.ipv4.udp_l3mdev_accept=1
-net.core.rmem_max=2097152
+net.core.rmem_max=3145728
 net.core.wmem_max=2097152
```

Comment:
do we need to update the wmem?

Reply:
I think we can increase both values

Reply:
We can increase the send buffer size to 3M. Updated PR with the changes.